spec/factories/diagnoses_factory.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+FactoryGirl.define do
+  factory :diagnosis, class: Debug do
+    association :file_version
+    association :file_range
+    kind { [Error, Warn, Hint, Debug].map(&:to_s).sample }
```

Comment:
Nitpick: `[Error, Warn, Hint, Debug].sample.to_s` might be slightly faster :grin: